Libraries/Opc.Ua.PubSub/Opc.Ua.PubSub.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Opc.Ua.PubSub</AssemblyName>
```

Comment:
need to add the libs to list of to be signed assemblies release/debug

Reply:
added in _signlistRelease.txt_ and _signlistDebug.txt_ . I hope that's all..